src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java

```diff
@@ -488,29 +456,101 @@ public String getQuestionResultStatisticsCsv(
             if (hiddenRecipients.contains(recipient)) {
                 continue;
             }
-
-            String recipientTeam = bundle.getTeamNameForEmail(recipient);
-            boolean isRecipientGeneral = recipient.equals(Const.GENERAL_QUESTION);
-
-            Double averageScoreExcludingSelf = averageExcludingSelf.get(recipient);
-            String averageScoreExcludingSelfText =
-                    getAverageExcludingSelfText(showAvgExcludingSelf, df, averageScoreExcludingSelf);
-
-            csvBody.append(SanitizationHelper.sanitizeForCsv(recipientTeam) + ','
-                           + SanitizationHelper.sanitizeForCsv(isRecipientGeneral
-                                                      ? "General"
-                                                      : bundle.getNameForEmail(recipient))
-                           + ','
-                           + df.format(average.get(recipient)) + ','
-                           + df.format(min.get(recipient)) + ','
-                           + df.format(max.get(recipient))
-                           + (showAvgExcludingSelf ? ',' + averageScoreExcludingSelfText : "")
-                           + Const.EOL);
+            updateQuestionResultStatisticsCsv(csvBody, bundle, recipient, average,
+                    min, max, showAvgExcludingSelf, averageExcludingSelf);
         }
 
         return csvHeader + csvBody.toString();
     }
 
+    /** adds a single record to the QuestionResultStatisticsCsv.
+     * @return void
+     */
+    private void updateQuestionResultStatisticsCsv(StringBuilder csvBody, FeedbackSessionResultsBundle bundle,
+            String recipient, Map<String, Double> average, Map<String, Double> min,
+            Map<String, Double> max, boolean showAvgExcludingSelf, Map<String, Double> averageExcludingSelf) {
+
+        String recipientTeam = bundle.getTeamNameForEmail(recipient);
+        boolean isRecipientGeneral = recipient.equals(Const.GENERAL_QUESTION);
+
+        DecimalFormat df = customDecimalFormat(0, 5, "down");
+
+        Double averageScoreExcludingSelf = averageExcludingSelf.get(recipient);
+        String averageScoreExcludingSelfText =
+                getAverageExcludingSelfText(showAvgExcludingSelf, df, averageScoreExcludingSelf);
+
+        csvBody.append(SanitizationHelper.sanitizeForCsv(recipientTeam) + ','
+                + SanitizationHelper.sanitizeForCsv(isRecipientGeneral
+                    ? "General"
+                    : bundle.getNameForEmail(recipient))
+                + ','
+                + df.format(average.get(recipient)) + ','
+                + df.format(min.get(recipient)) + ','
+                + df.format(max.get(recipient))
+                + (showAvgExcludingSelf ? ',' + averageScoreExcludingSelfText : "")
+                + Const.EOL);
+    }
+
+    /** creates a DecimalFormat with a given minimum & maximum fraction digits, and a rounding mode.
+     * @return DecimalFormat
```

Comment:
- Add a line break after the first line of the javadoc, i.e. after `/**`
- creates -> Creates (capitalize)
- Remove the `@return` tag since it does not value-add (one can tell that the return type is `DecimalFormat` by just looking at the method itself)